src/test/java/net/snowflake/client/jdbc/SnowflakeDriverIT.java

```diff
@@ -786,6 +786,7 @@ public void testPutWithWildcardGCP() throws Throwable {
     String destFolderCanonicalPathWithSeparator = destFolderCanonicalPath + File.separator;
 
     try {
+      statement.execute("alter session set ENABLE_GCP_PUT_EXCEPTION_FOR_OLD_DRIVERS=false");
```

Comment:
Will the customers run into this problem if they use old driver?

Reply:
resolved offline

Reply:
Yes. This change to inspire them to upgrade!!!